api/rbac/v1alpha1/annotations.go

```diff
@@ -15,6 +22,17 @@ const (
 	// may be either a scalar string value or a comma-separated list.
 	AnnotationKeyOIDCClaimNamePrefix = "rbac.kargo.akuity.io/claim."
```

Comment:
I want to stop short of deprecating it, because I just don't see a real need to do so, but could we add a comment here indicating that a string representation of a JSON object in the AnnotationKeyOIDCClaims is the preferred method of doing this going forward?